home/templates/home/explore.html

```diff
@@ -0,0 +1,75 @@
+{% extends "home/base.html" %}
+{% load static %}
+{% block content %}
+
+<style>
+
+.subject-img{
+	float:left;
+	width: 120px;
+	height: 100px;
+	margin: 15px;
+}
+
+.question-display{
+  background-color: #f3ebe6;
+}	
+
+.shlifim-btn{
+  background-color: #A9D4D9;
+}
+
+.landing-bg{
+	background-color: #f2dccb;
+}
+
+.collapsed-bg
+{
+    background-color: #f3ebe6;
+}
+
+</style>
```

Comment:
I wrote all the style in the HTML also, but Barak suggest that we move it do a different CSS file in the Static folder

Reply:
I moved it to different css file as requested